ppinot-model/src/main/java/es/us/isa/ppinot/evaluation/logs/LogEntry.java

```diff
@@ -57,6 +57,12 @@ public LogEntry(String processId, String instanceId, String bpElement, ElementTy
         this(processId, instanceId, bpElement, elementType, eventType, timeStamp);
         this.data.putAll(data);
     }
+    
+    public LogEntry(String processId, String instanceId, String bpElement) {
```

Comment:
Is this really necessary? It seems it is only used by method test, which is not used elsewhere.